### ISSUE TITLE
User Feedback missing

### ISSUE THREAD

**Issue by dennishenry** [getsentry/sentry#3115]
Version: 8.3.2

I apologize in advance if this is the incorrect way to report this issue.

I recently stumbled upon the [Blog Post](http://blog.getsentry.com/2016/04/21/introducing-user-feedback.html) announcing User Feedback, and integrated it into my project's staging environment. The process works as expected, popping up the JS box and allowing the user to report their issue, and the POST is then received by the server, but for the life of me, I cannot find that data. Is there some flag I need to enable on my server to accept/show the feedback?

Thank you!

**Comment by mattrobenolt** [comment]
It'll be an led by default in 8.4 which will be released next week. Probably should have mentioned that in blog post.

For 8.3, you can enable it by setting: `SENTRY_FEATURES['projects:user-reports'] = True` in `sentry.conf.py`.

**Comment by dennishenry** [comment]
@mattrobenolt Awesome thank you! I can see the tab now and everything seems on the level, but the reports I submit are still not showing up, despite the successful POST I'm seeing in the logs:

```
2016-04-25 10:16:32,475 DEBG 'sentry-web' stdout output:
127.0.0.1 - - [25/Apr/2016:14:16:32 +0000] "POST /api/embed/error-page/?eventId=%7B%7B%20bf99b62e62c147a598df4b4375afc6f5%20%7D%7D&dsn=[REDACTED VALID DSN] HTTP/1.0" 200 343 "[REDACTED APPLICATION URL]" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.86 Safari/537.36"
```

Is there anything else I'm missing here? Thanks again for this awesome feature!

**Comment by mattrobenolt** [comment]
Hmm, there shouldn't be anything you're missing, but I'm not entirely sure to be honest. It's very likely that there were bugs in 8.3 since it was an experimental feature, and they've all been fixed on master for 8.4, so I'd just wait until then, and if it's still problematic, we can address.
